api/src/services/uploads/uploads.ts

```diff
@@ -7,12 +7,32 @@ import type {
 
 import { s3PutSignedUrl } from 'src/lib/aws'
 import aws from 'src/lib/aws'
+import { ROLES } from 'src/lib/constants'
 import { db } from 'src/lib/db'
 import { logger } from 'src/lib/logger'
 import { ValidationError } from 'src/lib/validation-error'
 
 export const uploads: QueryResolvers['uploads'] = () => {
-  return db.upload.findMany()
+  const currentUser = context.currentUser
+
```

Comment:
I'd like to make a slight clarification to the logic. The `organizationId` check should be done for all users regardless of whether they are staff. Only staff should have the added check of seeing only files they have uploaded. Here's what I'm thinking-

```js
const whereInputs = {
    agency: { organizationId: currentUser.agency.organizationId }
}

if (hasRole([ROLES.ORGANIZATION_STAFF])) {
    whereInputs.uploadedById = currentUser.id
}

return db.upload.findMany( { where: whereInputs } )
```

Reply:
@as1729 Adjusted the logic (and tests), thanks for the feedback!